src/query/interpreter.cpp

```diff
@@ -2385,6 +2398,9 @@ PreparedQuery PrepareProfileQuery(ParsedQuery parsed_query, bool in_explicit_tra
   auto evaluator = PrimitiveLiteralExpressionEvaluator{evaluation_context};
   const auto memory_limit = EvaluateMemoryLimit(evaluator, cypher_query->memory_limit_, cypher_query->memory_scale_);
 
+  // TODO: how to expose metrics when we have hops limit?
+  const auto hops_limit = EvaluateHopsLimit(evaluator, cypher_query->hops_limit_);
```

Comment:
can you elaborate this question, I don't understand it at the moment?

Reply:
I think that comment isn't relevant anymore. Something that bothered me before -> should have delete it before review.